CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Unreleased
 
+## 1.197.0 (Nov, 13 2024)
+ENHANCEMENTS:
+* resource/spotinst_ocean_right_sizing_rule: Fixed rightsizing rule attach, detach issue for duplicate workloads. 
```

Comment:
correct the statement to Fixed rightsizing rule attach and detach workloads in subsequent updates

Reply:
Updated